site/docs/flink.md

```diff
@@ -97,7 +97,21 @@ CREATE CATALOG hive_catalog WITH (
   'catalog-type'='hive',
   'uri'='thrift://localhost:9083',
   'clients'='5',
-  'property-version'='1'
+  'property-version'='1',
+  'hive-conf-dir'='/opt/hive/conf'
+);
+```
+
+Alternatively one can instead set just the `warehouse` property (without specifying a Hive configuration directory) to initialize the Hive catalog:
```

Comment:
A small clarification: this implies that the use of `hive-site.xml` and `warehouse` are alternatives, but they aren't. You may want to use `warehouse` from the Hive site, or you may want to use other settings from `hive-site.xml` with a custom warehouse location. I think this should document the two options separately.

For documenting options, I think we will want to add a table because copying the SQL block each time creates a long doc. I would have one simple example for each catalog type with the options we recommend for it (Hive: uri, warehouse, and property-version; Hadoop: warehouse, property-version) and a table of other options for the Hive catalog to document `clients` and `hive-conf-dir`.

Reply:
Well, let me make this more clear.